### PR TITLE
Add agent status datatype into Ingestc.

### PR DESCRIPTION
**Pull request by kkuzmin**
### Solution Description
Add `agentstatus` support for collectors to be able to report their status via Ingest

### CODE REVIEW

File: ingestc.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                'x-invoked-by' : this._functionType,
+                'Content-Encoding' : 'deflate',
+                'Content-Length' : Buffer.byteLength(data)
+            },
+            body : data
+        };
+        return this.post(`/data/vpcflow`, payload);
+    }
+    
+    sendAicspmsgs(data) {
```
Comment (JDragovichAlertLogic) [inline ingestc.js]: Do we still need this one in here? since this is going into the NPM branch?

File: ingestc.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                'x-invoked-by' : this._functionType,
+                'Content-Encoding' : 'deflate',
+                'Content-Length' : Buffer.byteLength(data)
+            },
+            body : data
+        };
+        return this.post(`/data/secmsgs`, payload);
+    }
+
+    sendVpcFlow(data) {
```
Comment (JDragovichAlertLogic) [inline ingestc.js:45]: As far as I know, this message type doesn't exist? Should we remove it? If we do VPC flow logs in the future, won't they be `logmsgs`?

**Review by JDragovichAlertLogic — CHANGES_REQUESTED** [review]
Thanks Kanstatnsin. A few comments.

File: ingestc.js (hunk 2 above)
Comment (kkuzmin) [reply ingestc.js:45]: it still exists. We may want to reuse it.

Comment (JDragovichAlertLogic) [reply ingestc.js:45]: cool. Lets keep it then

**Review by JDragovichAlertLogic — APPROVED** [review]
LGTM. thanks Kanstantsin

**Pull request merged**